common/bytes.go

```diff
@@ -24,6 +24,13 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 )
 
+const (
+	Kilobytes = 1024
```

Comment:
Kind of a nit, but I'd rename these to be singular instead of plural (feels more inline with the Go stdlib, e.g. time unit constants defined in the `time` package).